<a href="url"><img src="public/yuttapong.png" align="left" height="150"></a>

<a id="top"></a>

#### Research Fellow, [Yang Lab](http://abacus.gene.ucl.ac.uk/)

#### [Department of Genetics, Evolution and Environment (GEE)](https://www.ucl.ac.uk/biosciences/gee), University College London, UK

[UCL profile page](https://profiles.ucl.ac.uk/45794-yuttapong-thawornwattana)

üì¨ gmail.com or ucl.ac.uk: yuttapong.thawornwattana


## Hi there üëã

- ü¶ü [Currently working on](#current-projects--) the phylogeny and evolution of _Anopheles_ mosquitoes
- üêù Currently learning: genome assembly, infectious disease epidemiology, malaria
- üêõ Looking to collaborate on mosquito genomics, phylogeny and evolution, inference of gene flow from genomic data

---

## Main content

- üå± [Training & Research Experience](#training--research-experience--), ü™∏ [Bio](#bio--)
- ü™≤ [Research](#research--),  üêå [Current projects](#current-projects--)
- üê≥ [Publications](#publications--), [Google Scholar](https://scholar.google.com/citations?user=DjlTi4gAAAAJ), 
[NCBI Bibliography](https://www.ncbi.nlm.nih.gov/myncbi/yuttapong.thawornwattana.3/bibliography/public/), [ORCID](https://orcid.org/0000-0003-2745-163X), [Web of Science](https://www.webofscience.com/wos/author/record/1865943)
- üí¨ [Talks](#talks--), üêõ [Tutorials](#tutorials--), üêô [On Github](#on-github--)

---

## Training & Research Experience  z

- üî© 2024-2026\.&emsp; Research Fellow, [UCL Department of Genetics, Evolution and Environment (GEE)](https://www.ucl.ac.uk/biosciences/gee), UK (with [Ziheng Yang](http://abacus.gene.ucl.ac.uk/))
- ü¶ã 2020-2024\.&emsp; PhD [Organismic and Evolutionary Biology (OEB)](https://oeb.harvard.edu/), Harvard University, USA (with [James Mallet](https://mallet.oeb.harvard.edu/) and [Ziheng Yang](http://abacus.gene.ucl.ac.uk/)), with the Interdisciplinary Concentration in Infectious Disease Epidemiology ([ID Epi](https://www.hsph.harvard.edu/idepi/about/))
- ü¶† 2018-2020\.&emsp;Research assistant, [Center for Microbial Genomics (CENMIG)](https://microbiology.sc.mahidol.ac.th/cenmig/), Department of Microbiology, Faculty of Science, Mahidol University, Thailand
- üß¨ 2014-2018\.&emsp;MPhil Genetics, Evolution and Environment, [UCL GEE](https://www.ucl.ac.uk/biosciences/gee), UK (with [Ziheng Yang](http://abacus.gene.ucl.ac.uk/))
- üõ† 2012-2013\.&emsp;MSc Computational Statistics and Machine Learning, [UCL Computer Science](https://www.ucl.ac.uk/computer-science/), UK
- üêõ 2009-2012\.&emsp;BSc Biological Sciences, [UCL GEE](https://www.ucl.ac.uk/biosciences/gee), UK


---

## Bio  [‚§¥Ô∏è](#top)

I am a postdoc in Ziheng Yang's group at UCL working on statistical methods for studying introgression from genomic data.  

I obtained my BSc in Biological Sciences from UCL focusing on phylogenetics and molecular evolution, and MSC in Computational Statistics and Machine Learning, also from UCL, focusing on Bayesian statistics and Monte Carlo methods.  I then went on to do my PhD with James Mallet at Harvard University, studying introgression in _Heliconius_ butterflies and _Anopheles_ mosquitoes.  I also studied infectious disease epidemiology at the Harvard T.H. Chan School of Public Health.

---

## Research  [‚§¥Ô∏è](#top)

How do insects become vectors of human diseases?  Why are some species better vectors than others?  How are they adapting to changing environments and what does it mean for transmission of vector-borne diseases?  

To answer these questions, I use malaria mosquitoes in the genus _Anopheles_ as model systems.  They are useful for studying not only fundamental questions in evolutionary biology about species divergence and speciation but also practical questions in infectious disease biology and epidemiology ranging from vector surveillance to global health and disease eradication.

More broadly, I study how populations and species evolve, and how traits of medical and epidemiological interests arise and change over time using genomic data and statistical approaches.  

Research areas: phylogenetics, evolutionary genomics, mosquito evolution, malaria, computational statistics

---


## Publications  [‚§¥Ô∏è](#top)

[Google Scholar profile](https://scholar.google.com/citations?user=DjlTi4gAAAAJ), 
[NCBI Bibliography](https://www.ncbi.nlm.nih.gov/myncbi/yuttapong.thawornwattana.3/bibliography/public/)

- __Thawornwattana Y__, Flouri T, Mallet J, Yang Z. (2024) __Inference of continuous gene flow between species under misspecified models__. bioRxiv [[doi:10.1101/2024.05.13.593926](https://doi.org/10.1101/2024.05.13.593926)] [[C code](https://github.com/ythaworn/iimmsci2s)]
- Mallet J, Seixas FA, __Thawornwattana Y__ (2024). __Species, Concepts of.__ In Scheiner, S. M., editor, _Encyclopedia of Biodiversity (Third Edition)_, 7:531-545. Elsevier. [[doi:10.1016/B978-0-12-822562-2.00022-0](https://doi.org/10.1016/B978-0-12-822562-2.00022-0)] [[RG](https://www.researchgate.net/publication/358303271_Species_Concepts_of)]
- __Thawornwattana Y__, Seixas FA, Yang Z, Mallet J. (2023) __Major patterns in the introgression history of _Heliconius_ butterflies__. _eLife_. [[doi:10.7554/eLife.90656](https://doi.org/10.7554/eLife.90656)] [[data](https://zenodo.org/records/8415106)]
- __Thawornwattana Y__, Huang J, Flouri T, Mallet J, Yang Z. (2023) __Inferring the direction of introgression using genomic sequence data__. _Mol Biol Evol._, 40(8):msad178. [[doi:10.1093/molbev/msad178](https://doi.org/10.1093/molbev/msad178)] [[_Heliconius_ data](https://zenodo.org/records/8243142)]
- Huang J, __Thawornwattana Y__, Flouri T, Mallet J, Yang Z. (2022) __Inference of gene flow between species under misspecified models__. _Mol Biol Evol._, 39(12):msac237. [[doi:10.1093/molbev/msac237](https://doi.org/10.1093/molbev/msac237)] [[C code](https://github.com/ythaworn/iimmsci2s)]
- __Thawornwattana Y__, Seixas FA, Yang Z, Mallet J. (2022) __Full-likelihood genomic analysis clarifies a complex history of species divergence and introgression: the example of the erato-sara group of _Heliconius_ butterflies.__ _Syst Biol._, 71(5):1159‚Äì1177. [[doi:10.1093/sysbio/syac009](https://doi.org/10.1093/sysbio/syac009)] [[data](https://zenodo.org/records/5639716)]
- Aiewsakun P, Ruangchai W, __Thawornwattana Y__, Jaemsai B, Mahasirimongkol S, Homkaew A, Suksomchit P, Dubbs P, Palittapongarnpim P. (2022) __Genomic epidemiology of _Streptococcus agalactiae_ ST283 in Southeast Asia.__ _Sci Rep._, 12(1):4185. [[doi:10.1038/s41598-022-08097-0](https://doi.org/10.1038/s41598-022-08097-0)]
- Netikul T, __Thawornwattana Y__, Mahasirimongkol S, Yanai H, Maung HMW, Chongsuvivatwong V, Palittapongarnpim P. (2022) __Whole-genome single nucleotide variant phylogenetic analysis of _Mycobacterium tuberculosis_ lineage 1 in endemic regions of Asia and Africa.__ _Sci Rep._, 12(1):1565. [[doi:10.1038/s41598-022-05524-0](https://doi.org/10.1038/s41598-022-05524-0)] [[phylogeny: Asia/Africa/Oceania](https://microreact.org/project/ii9NYi9YWaWFjZkivV2JoC); [phylogeny: global 1](https://microreact.org/project/m8QSwp8XhXBUhLTDhRuGB9); [phylogeny: global 2](https://microreact.org/project/rtJuewY8AXZFZLKVRQXc3H)]
- Netikul T, Palittapongarnpim P, __Thawornwattana Y__, Plitphonganphim S. (2021) __Estimation of the global burden of _Mycobacterium tuberculosis_ lineage 1.__ _Infect Genet Evol._, 91:104802. [[doi:10.1016/j.meegid.2021.104802](https://doi.org/10.1016/j.meegid.2021.104802)]
- __Thawornwattana Y__, Dong HT, Phiwsaiya K, Sangsuriya P, Senapin S, Aiewsakun P. (2021) __Tilapia lake virus (TiLV): Genomic epidemiology and its early origin.__ _Transbound Emerg Dis._, 68(2):435‚Äì444. [[doi:10.1111/tbed.13693](https://doi.org/10.1111/tbed.13693)]
- __Thawornwattana Y__, Mahasirimongkol S, Yanai H, Maung HMW, Cui Z, Chongsuvivatwong V, Palittapongarnpim P. (2021) __Revised nomenclature and SNP barcode for _Mycobacterium tuberculosis_ lineage 2.__ _Microb Genom._, 7(11):000697. [[doi:10.1099/mgen.0.000697](https://doi.org/10.1099/mgen.0.000697)] [[phylogeny: discovery](https://microreact.org/project/4P2iPeBx1Y66TyJfojNM1o); [phylogeny: discovery+test](https://microreact.org/project/4kJjFVPWe3TbfwFtqY3ED8)]
- Ajawatanawong P, Yanai H, Smittipat N, Disratthakit A, Yamada N, Miyahara R, Nedsuwan S, Imasanguan W, Kantipong P, Chaiyasirinroje B, Wongyai J, Plitphonganphim S, Tantivitayakul P, Phelan J, Parkhill J, Clark TG, Hibberd ML, Ruangchai W, Palittapongarnpim P, Juthayothin T, __Thawornwattana Y__, Viratyosin W, Tongsima S, Mahasirimongkol S, Tokunaga K, Palittapongarnpim P. (2019) __A novel Ancestral Beijing sublineage of _Mycobacterium tuberculosis_ suggests the transition site to Modern Beijing genotypes.__ _Sci Rep._, 9:13718. [[doi:10.1038/s41598-019-50078-3](https://doi.org/10.1038/s41598-019-50078-3)]
- __Thawornwattana Y__, Dalquen D, Yang Z. (2018) __Coalescent analysis of phylogenomic data confidently resolves the species relationships in the _Anopheles_ gambiae species complex.__ _Mol Biol Evol._, 35(10):2512‚Äì2527. [[doi:10.1093/molbev/msy158](https://doi.org/10.1093/molbev/msy158)]
- __Thawornwattana Y__, Dalquen D, Yang Z. (2018) __Designing simple and efficient Markov chain Monte Carlo proposal kernels.__ _Bayesian Anal._, 13(4):1037‚Äì1063. [[doi:10.1214/17-BA1084](https://doi.org/10.1214/17-BA1084)]
- dos Reis M, __Thawornwattana Y__, Angelis K, Telford MJ, Donoghue PC, Yang Z. (2015). __Uncertainty in the timing of origin of animals and the limits of precision in molecular timescales.__ _Curr Biol._, 25(22):2939‚Äì2950. [[doi:10.1016/j.cub.2015.09.066](https://doi.org/10.1016/j.cub.2015.09.066)] [[data](https://doi.org/10.6084/m9.figshare.1525089.v1)]

---

## Talks  [‚§¥Ô∏è](#top)

- 2024/10\.&emsp;"Evolutionary genomics of mosquitoes" at the 5th International Course on Ecological Determinants of Vector-Borne Disease Dynamics, Reserva Ecol√≥gica de Guapia√ßu (REGUA), Rio de Janeiro, Brazil [[slides](https://drive.google.com/file/d/1nS7_fIeUfWSZS0xZfCiiVauPGIZQSSOp/view?usp=sharing)]
- 2024/09\.&emsp;"Evolutionary genomics of _Anopheles_ mosquitoes" at the Department of Parasitology, Faculty of Medicine, Chiang Mai University [[slides](https://drive.google.com/file/d/1nW3JVpFKoyN3qrBMmoDjKxXSmU0Yp9sl/view?usp=sharing)]
- 2024/09\.&emsp;"Evolutionary genomics of malaria mosquitoes" at Faculty of Tropical Medicine, Mahidol University [[slides](https://drive.google.com/file/d/1OR0HFvePZSFAO8hJLg_b1JBLvMtcWUwc/view?usp=sharing)]
- 2024/07\.&emsp;"Inferring species phylogeny and gene flow from genomic data", a dissertation seminar at Harvard OEB [[slides](https://drive.google.com/file/d/1Womh4FuREXZ8NE4Aco5Fjb1e1lci7niv/view?usp=sharing)]
- 2024/04\.&emsp;"Evolution of _Anopheles_ mosquitoes in North America" in G4 Symposium at Harvard OEB
- 2023/09\.&emsp;"Inferring the direction of gene flow from genomic data" at [Boston Evolutionary Genomics Supergroup](https://evogen.hms.harvard.edu) [Annual Retreat 2023](https://evogen.hms.harvard.edu/annual-retreat-2023), the Broad Institute (chalk talk)
- 2023/06\.&emsp;"Inferring phylogenies using full-likelihood multispecies coalescent with introgression" at [Evolution conference](https://www.evolutionmeetings.org/) (Albuquerque, New Mexico, USA) [[slides](https://drive.google.com/file/d/1oMPWP17bdJN8xcvGi3JSY2WQkWnxycIY/view?usp=sharing), [with builds](https://drive.google.com/file/d/1myaH0pXqBabqxToTtiuwvga1ixW749v4/view?usp=sharing)] [[recording](https://youtu.be/sz9du0CxDkk?si=4qz_lTDIHoGOUDs7&t=2642)]
- 2021/06\.&emsp;"Full-likelihood analysis of divergence and introgression in the erato-sara group of _Heliconius_" at [Heliconius Seminar](https://www.heliconius.org/lockdown-seminars/) (now Lepinars) [[slides](https://drive.google.com/file/d/1SPH6DcDR3AUyQyanLVUbmAEtI4_JaSKz/view?usp=drive_link)]
- 2020/12\.&emsp;"Whole-genome sequencing for strain typing in practice" at [CENMIG](https://microbiology.sc.mahidol.ac.th/cenmig/) Workshop, Mahidol University [[slides](https://drive.google.com/file/d/1Dgsjs_Q-ZdI_BPvCtiK8zYOQ2d63RDh6/view?usp=sharing)]
- 2020/11\.&emsp;"Population genomics of _Mycobacterium tuberculosis_" at the Department of Microbiology, Faculty of Science, Mahidol University [[slides](https://drive.google.com/file/d/1pCzQJurTjpLYdC0wLQgps2K_gRyT1ivw/view?usp=sharing)]
- 2020/07\.&emsp;"Population genomics of _Mycobacterium tuberculosis_" at [MAEMOD](https://www.tropmedres.ac/units/moru-bangkok/mathematical-and-economic-modelling) journal club [[slides](https://drive.google.com/file/d/14l6LsTVUvFhiMeessBPRvgmNXmN9JgeG/view?usp=sharing)]
- 2018/11\.&emsp;"Next generation phylogenetics: Building trees from whole genome data" at the Division of Medical Bioinformatics, Faculty of Medicine Siriraj Hospital, Mahidol University [[slides](https://drive.google.com/file/d/1M4aeZWZG1VW_6utC8oRFsXJLYOPYUXOX/view?usp=sharing)]
- 2018/08\.&emsp;"Resolving the species relationships in the _Anopheles gambiae_ species complex from whole genome data" at the Department of Microbiology, Faculty of Science, Mahidol University [[slides](https://drive.google.com/file/d/111TM768NdjTm1uYrwh6IQmlLD4MDeeW4/view?usp=sharing)]

---

## On Github  [‚§¥Ô∏è](#top)

* [`iimmsci2s`](https://github.com/ythaworn/iimmsci2s): Calculate asymptotic maximum likelihood estimates, also known as pseudo-true parameter values, of parameters under the multispecies coalescent model with pulse introgression or continuous migration
* [`make-3s-data`](https://github.com/ythaworn/make-3s-data): Phase multilocus data and sample 3 sequences per locus for 3s analysis
* [`mtbtyper`](https://github.com/ythaworn/mtbtyper): Genotype strains of _Mycobacterium tuberculosis_ isolates from whole-genome sequence data
* [`group-specific-variants`](https://github.com/ythaworn/group-specific-variants): Identify group-specific single-nucleotide variants from multiple sequence alignment
* [`dvrcaller`](https://github.com/ythaworn/dvrcaller): Delect DVR (direct variable repeat) deletion pattern in the CRISPR region of _Mycobacterium tuberculosis_ using whole-genome sequence data
* [`snpplet`](https://github.com/ythaworn/snpplet): [Nextflow](https://www.nextflow.io/) pipeline for short variant calling (SNPs) from paired-end short-read genome sequencing data

---

## Tutorials  [‚§¥Ô∏è](#top)

-  Bayesian linear regression (2018) - [slides](https://drive.google.com/file/d/1UnN1ZU3x6KsCyKU4Ck_AsWozVu_CR10S/view?usp=sharing)
-  Dirichlet process mixture (DPM) model (2018) - [slides](https://drive.google.com/file/d/1NPN_WF7YWXZFLkUjvIhJH10roRAm81uE/view?usp=sharing)
-  [TensorFlow tutorial (2016)](https://www.homepages.ucl.ac.uk/~zcbtfu5/)

---

## Current projects  [‚§¥Ô∏è](#top)

- Phylogeny and evolution of _Anopheles_ mosquitoes (Diptera: Culicidae: Anophelinae) in North America: [specimens](https://ythaworn.github.io/public/anopheles/specimens.html)
- Methods for inferring gene flow from genomic data
- Bioinformatics pipeline for generating multilocus data from short-read sequencing data
- Introgression histories in _Heliconius_ butterflies (Lepidoptera: Nymphalidae: Heliconiinae: Heliconiini)
- Taxonomic catalogue of Heliconiinae (Lepidoptera: Nymphalidae)

---
